src/osd/ReplicatedPG.cc

```diff
@@ -3650,12 +3650,35 @@ static int check_offset_and_length(uint64_t offset, uint64_t length, uint64_t ma
   return 0;
 }
 
-struct FillInExtent : public Context {
+struct FillInVerifyExtent : public Context {
```

Comment:
[OT] note to myself, i will need to update the implementation `ToSparseReadResult` to do something alike after this pr is merged.